fixture/app.js

```diff
@@ -9,16 +9,68 @@
 const http = require('http')
 const express = require('express')
 const bodyParser = require('body-parser')
+const path = require('path')
+const ejs = require('ejs')
+const urllib = require('urllib')
+const url = require('url')
+var qs = require('querystring')
 
 const course = require('./api/course')
 const ping = require('./api/ping')
 
 const app = express()
+// 使用ejs模板
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'ejs');
 
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: false }))
+app.use(express.static(path.join(__dirname, 'dist')))
 
 app.use('/api', ping())
 app.use('/api', course())
 
+// 首页全部课程列表
+app.get('/', function(req, res, next) {
+  urllib.request(req.headers.host + '/api/courses', {dataType: 'json'}, function (err, data, result) {
+    if (err) {
+      throw err
```

Comment:
`Error` here should be caught, otherwise it will crash the server. 
See more http://expressjs.com/en/guide/error-handling.html